<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendfiledialog</class>
 <widget class="QWidget" name="sendfiledialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send File</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>820</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/background/pic/background/fileBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="userName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#3E5684</string>
   </property>
   <property name="text">
    <string>Send File</string>
   </property>
  </widget>
  <widget class="QLabel" name="userName_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#3E5684</string>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QLabel" name="friendName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>581</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#3E5684</string>
   </property>
   <property name="text">
    <string>Someone</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>581</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#3E5684</string>
   </property>
   <property name="text">
    <string>请选择文件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseFileBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/pic/icon/chooseFile.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sendFileBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>560</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/pic/icon/sendFile.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="userName_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#3E5684</string>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="userName_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>650</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#3E5684</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
